demo/src/main.cpp

```diff
@@ -1,23 +1,33 @@
 #include <exception>
 #include <iostream>
+#include <thread>
+
+#include <arbitration_graphs/gui/web_server.hpp>
 
 #include "demo/pacman_agent.hpp"
 #include "demo/types.hpp"
 #include "utils/pacman_wrapper.hpp"
 
 using namespace demo;
 using namespace utils;
+using namespace arbitration_graphs;
 
 int main() {
     try {
         PacmanWrapper demo;
         PacmanAgent agent(demo.game());
 
+        gui::WebServer server(8080, true);
+        server.loglevel(crow::LogLevel::Info);
```

Comment:
I'd set this to `Warning`, right? There's quite a bit of spam in the terminal right now that would not really be necessary.